CONTRIBUTING.md

```diff
@@ -350,6 +353,38 @@ This involves the following steps:
    ```bash
    >>> git push origin --delete feature-add-new-important-headers
    ```
+## Testing the documentation
+
+`oomph-lib`'s extensive documentation is generated by doxygen and the full set of webpages (containing many detailed tutorials) is hosted on Github. The webpages are regenerated automatically whenever the `main` branch of the official `oomph-lib` repository is updated. It is therefore important that you check any changes to the documentation and make sure that they render correctly before they are ultimately committed to the main branch. It is therefore important that you check any changes to the documentation and make sure that they render correctly before they are ultimately committed to the `main` branch.
+
+For this purpose, you can manually trigger the webpage-regeneration workflow from within your forked `oomph-lib` repository. This process overwrites the version of the webpages associated with that forked repository rather than the official ones. This requires the following steps:
+
+### Initial setup (only to be done once)
+
+Click on the "Settings" tab in Github webpage of your forked repository:
```

Comment:
- "Github" -> "GitHub"
- "tab in Github webpage" -> "tab from the GitHub webpage"

Reply:
I've made a global change to GitHub and fixed the grammar.

  Done.